pkg/server/controller/start_ekadashi.go

```diff
@@ -0,0 +1,69 @@
+package controller
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/Lighty0410/ekadashi-server/pkg/ekadashi"
+	"github.com/Lighty0410/ekadashi-server/pkg/mongo"
+)
+
+// FillEkadashi is a goroutine that autofill ekadashi dates via getEkadashi().
+// If succeed it adds ekadashi date to the database.
+// It gonna logs errors only. Not to return themselves.
+func (c *Controller) FillEkadashi(ctx context.Context) {
+	go func() {
+		actualEkadashi, err := c.getEkadashi()
+		if err != nil {
+			log.Println(err)
```

Comment:
If you are unable to fill for the first time this means server is down and cannot be started